server/src/main/java/io/crate/metadata/pgcatalog/OidHash.java

```diff
@@ -41,34 +48,55 @@
         PROC
     }
 
+    private static int oid(String... keys) {
```

Comment:
Due to the varargs this results in an array allocation. I think in most call-sites we could just go for string concat using ` +` and then do the `.getBytes` call to avoid some overhead.